### DISCUSSION TITLE
FIXED: Regenerated personal access token, what now?

### DISCUSSION THREAD

**Discussion by ccccc170** [community/community#31639] [New to GitHub]
EDIT: I Have fixed this now. It was actually working as normal apart from the remote's urls had somehow changed to become 2x what they were supposed to be in the repos I was trying to push. This seemingly happened when trying to push the first time with the token reset but I'm not certain. Bizzare and no idea what happened but it's fine now. Can't see a way to delete this post.

Hello,

I recently recieved an email saying that the personal access token I set up when I first started using gitub was about to expire. Upon recieving this email I followed the link to regenerate the token however after doing this I can no longer push my repos to github and recieve an error message in my terminal stating `fatal: repository <repository-name-here> not found`. There is no further instruction provided in the error message nor on the token page itself or the emails I've recieved about updating my token that tells me how I set everything up so that my newly regenerated token allows for me to continue to push repos etc. I am not prompted to enter the new token. I searched online and cannot find any simple instructions regarding what I need to do now.

I have not been coding or using github for very long. I found setting up my intial token a farce when I did that and can't remember exactly how I did it. Could anyone help me out massively by providing me with some simple, straightfoward instructions about what I need to do now I have regenerated my token (I have saved the token from the regeneration page) in order to be able to continue using github (pushing etc.) as I have been doing with the repos I have been working on? Additionally is there another way I can set this up so I don't have to mess about with the token (I've seen that there's another method called SSH as well however I have been unable to find any instructions that are simple enough for me to understand and follow regarding that too)? I have absolutely no care at all about any security benefits this method brings, I'd just like to do something like enter a passsword once for all my repos and be done with it.

Thanks!

**Comment by airtower-luna** [comment]
The key here is that "regenerating" a token means creating a new one with the same permissions, so you have to replace the old token with the new one. Good thing you saved it! :smiley_cat: 

> I am not prompted to enter the new token.

That implies your old token is stored permanently in some kind of credential manager. Where to delete or replace that depends on the kind of credential manager you use. And that in turn depends on your OS or desktop environment.

You can check `git config --get credential.helper` to see which one(s) are enabled on your machine. If you share that and your OS, hopefully someone has an idea. :slightly_smiling_face:

**Reply by ccccc170** [reply]
I seem to have fixed this now, although I've no idea what happened. Thanks for your response though!
